### PR TITLE
Add vue viewer

### PR DESCRIPTION
**Pull request by eraulo**
Closes #no

## Description of the Problem / Feature
-  
## Explanation of the solution
- Added vue viewer.

## Instructions on making this work
- npm install
- npm run build
- ./manage.py runserver

### CODE REVIEW

**Comment by marobo** [comment]
I think we don't need zoom button, we can make it by just click on the image.

File: src/components/PhotoTimor.vue [hunk 1]
```diff
@@ -17,16 +16,12 @@
                         <p>
                             {{ $filters.shorten(image.istoria.description, 75) }}
                         </p>
-                        <span>Uploaded by {{ image.istoria.creator }}</span>
+                        <div><span>Uploaded by {{ image.istoria.creator }}</span></div>
+                        <button type="button" class="btn-outline-success" @click="selectImgObject(image.image)">Zoom</button>
```
Comment (marobo) [inline src/components/PhotoTimor.vue:20]: I think we don't need zoom button, we can make it by just click on the image.

File: src/components/PhotoTimor.vue [hunk 2]
```diff
@@ -91,6 +79,34 @@
...
+                        initialViewIndex: indexId,
+                        inline: true, 
+                        button: true, 
+                        navbar: true, 
+                        title: true, 
+                        toolbar: true, 
+                        tooltip: true, 
+                        movable: true, 
+                        zoomable: false, 
+                        rotatable: false,
```
Comment (marobo) [inline src/components/PhotoTimor.vue:101]: Have a rotatable icon button not working...

**Review by marobo — CHANGES_REQUESTED** [review]

File: src/components/PhotoTimor.vue (hunk 2 above)
Comment (eraulo) [reply src/components/PhotoTimor.vue:100]: This isn't rotatable icon only have prev button, next button and reset button.

Comment (marobo) [reply src/components/PhotoTimor.vue:100]: Screen Shot 2021-08-06 at 22 31 50

**Comment by marobo** [comment]
Error when I ran `npm run watch` in this branch!

**Comment by marobo** [comment]
> @eraulo , Error when I ran `npm run watch` in this branch!

Forgot to do `npm install`, Working fine now!

**Review by marobo — APPROVED** [review]

**Pull request merged**